src/services/getQuotes.service.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const axios = require('axios');
+const baseURL = 'https://type.fit/api/quotes';
+
+const getQuotes = async () => {
+    try {
+        // const response = await axios.get(`${baseURL}`);
+        // const completeQuoteArray = response.data;
+        return await axios.get(`${baseURL}`)
+    } catch (error) {
+        console.error(error);
+    }
+}
+
+const fetchFiftyQuote = async () => {
+    const completeQuotes = await getQuotes();
+    return completeQuotes.data.slice(0,50)
```

Comment:
How will another developer know, that the limit of 50 is defined in this particular in this line